api/src/main/java/org/pmiops/workbench/db/model/User.java

```diff
@@ -170,6 +171,17 @@ public void setAuthorities(Set<Authority> newAuthorities) {
     this.authorities = newAuthorities;
   }
 
+  @ElementCollection
+  @CollectionTable(name = "PageVisit", joinColumns=@JoinColumn(name="user_id"))
+  @Column(name="page_id")
+  public Set<PageVisit> getPageVisits() {
+    return pageVisits;
+  }
+
+  public void setPageVisits(Set<PageVisit> pageVisits) {
+    this.pageVisits = pageVisits;
```

Comment:
I may be changing this a little, as I'm not positive that it update page visits the way I want it to.